oci.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import logging
+
+from oci import config
+from oci import gerrit
+from oci import jenkins
+
+
+def main():
```

Comment:
If this file becomes ocu/main.py, a good name for this is run().

Reply:
Done.